django/contrib/admin/views/main.py

```
@@ -597,6 +597,13 @@ def apply_select_related(self, qs):
             return qs.select_related(*self.list_select_related)
         return qs
 
+    def get_search_results_unsafe(self, request, queryset, search_term):
```

Ruff (0.12.2)

Unused method argument: `request`

(ARG002)